pipeline/shell.py

```diff
@@ -333,6 +365,18 @@ def populate_psth(populate_settings={'reserve_jobs': True, 'display_progress': T
     psth.UnitSelectivity.populate(**populate_settings)
 
 
+def populate_foraging_psth(populate_settings={'reserve_jobs': True, 'display_progress': True}):
+
+    log.info('psth.UnitPsth.populate()')
+    psth_foraging.UnitPsth.populate(**populate_settings)
+
+    log.info('psth.PeriodSelectivity.populate()')
+    psth_foraging.PeriodSelectivity.populate(**populate_settings)
+
+    log.info('psth.UnitSelectivity.populate()')
+    psth_foraging.UnitSelectivity.populate(**populate_settings)
```

Comment:
These populate calls don't need to be in a separate function, can be included as part of the `populate_psth()` function.
If this is to be separated into `populate_foraging_psth`, then this should also be included in the `automate_computation` (further down) to be included in the automatic jobs processing by the workers
(Just a suggestion, no need to be in this PR, we can add/reconfigure in later PRs)

Reply:
Yes, it makes sense to include the population of foraging psth in the same `populate_psth()`. Will change it.